draft-ietf-quic-http.md

```diff
@@ -147,11 +149,16 @@ Origins SHOULD list only versions which are supported by the alternative, but
 MAY omit supported versions for any reason.
 
 
-# Connection Establishment {#connection-establishment}
+## Connection Establishment {#connection-establishment}
+
+HTTP/QUIC relies on QUIC as the underlying transport.  The QUIC version being
+used MUST use TLS version 1.3 or greater as its handshake protocol.  The Server
+Name Indication extension {{!RFC6066}} MUST be included in the TLS handshake.
```

Comment:
I missed the SNI reference because the acronym was not used, may help to add _(SNI)_.

RFC7540 also says clients MUST indicate target domain when negotiating TLS, is that requirement inherited here?

Reply:
I tried a couple of different wordings with the requirement on what to include in SNI, but nothing felt right.  (Particularly in light of the Alt-Svc SNI draft, which suggests that the SNI might be something to which you don't actually intend to direct resources.)  I ultimately decided to leave it at MUST use the extension and MUST make sure you've seen a valid cert for an origin before using the connection for it.  If those two are related or not is a case-by-case choice by the implementation.